projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
What is this file for? You don't implement any of these functions anywhere AFAICT.

Reply:
`lights_gpio` is another module that handles anything gpio related. From reading the gpio pin that says whether this is the front board or the back board, to turning peripherals on and off, etc. 
I had all these modules on one branch, then @tituschow told me to break them down into smaller pull requests. I need the definition of `BoardType` in the `lights_can` module (which is the module for this ticket/pull request), that's why I included this header from that branch. My next tickets will have the definitions of these functions, I can remove them here if you want.

Reply:
Yeah, please remove them, I'm pretty sure the build will break if stuff isn't defined. Just comment the call to them out or something until your next PR.

Reply:
there's no call to them right now, I'll comment the definitions out. too for now till my next PR